app/src/main/res/values/strings.xml

```diff
@@ -485,6 +485,15 @@
   <string name="home_screen_good_morning_greeting_fragment">Good morning,</string>
   <string name="home_screen_good_afternoon_greeting_fragment">Good afternoon,</string>
   <string name="home_screen_good_evening_greeting_fragment">Good evening,</string>
+  <!--  Privacy Policy -->
+  <string name="privacy_policy_activity_title">Privacy Policy</string>
+  <string name="privacy_policy">Privacy Policy.</string>
+  <string name="privacy_policy_web_link"><![CDATA[Please visit [<a href="https://www.oppia.org/privacy-policy">this page</a>] for the latest version of this policy]]></string>
+  <!--  Terms of Service -->
+  <string name="terms_of_service_activity_title">Terms of Service</string>
+  <string name="terms_of_service"> Terms of Service</string>
+  <string name="agree_to_terms">By using %s, you agree to our Terms of Service and Privacy Policy.</string>
```

Comment:
I am not understanding how to do this. Do you mean as below
 `<string name="agree_to_terms">By using %s, you agree to our <a href="#termsofservice">Terms of Service</a> and <a href="#privacypolicy">Privacy Policy</a>.</string>`

Reply:
Yes that's what I mean, though now that I think about it we can actually just create custom spans since we're using our own HtmlParser layer that lets us easily do this. Consider:

```
By using %s, you agree to our <link-to-terms-of-service>Terms of Service</link-to-terms-of-service> and <link-to-privacy-policy>Privacy Policy</link-to-privacy-policy>.
```

(With the angle brackets properly escaped).

WDYT? This seems much cleaner since it just means we need to implement (probably a single) custom tag handler and register it with both tags, and then just swap the tag with a clickable span that calls a listener. It'd work very similarly to how concept cards work today and simplify a lot of the onboarding binding complexity from above.

Reply:
I have created custom HTML tags for policies and a PolicyPageTagHandler to handle these tags. Everything seems correct but it is not triggering in OnboardingFragmentPresenter. Can you please check these two classing where I am going wrong.

Reply:
@veena14cs can you follow up with a debugging doc (https://github.com/oppia/oppia/wiki/Debugging-Docs)? I'm not sure I understand what exactly isn't working, and it's hard for me to provide guidance without understanding what you've already tried.

Reply:
Thanks @BenHenning 
 onPolicyPageLinkClicked() in OnboardingFragmentPresenter is not getting triggered when I click any of the Policy links.I will follow up with debugging doc with more details.

Reply:
I could figure out issue. I wasn't passing tag listener to htmlParserFactory(). Because of this it wasn't triggering onPolicyPageLinkClicked() in OnboardingFragmentPresenter.

Reply:
> @veena14cs I have found following issues with this PR:
> 
> 1. Have a look at RTL screenshots (a.) in mobile portrait it looks as if the sentence has been changed completelyed (b.) in mobile landscape there is `space` issue after `and`.
> 2. Select `ToS` in tablet portrait, rotate to landscape and you will notice on right side the header text is lost.
> 3. Check below screenshots in this check the edge text, it looks like it gets cut in both the screenshots. Same might apply to mobile too.
> 
> <img alt="" width="250" src="https://user-images.githubusercontent.com/9396084/140426869-be3762c6-ace1-4154-bfec-cb9734277752.png"> <img alt="" width="250" src="https://user-images.githubusercontent.com/9396084/140426874-57f6bcd2-e7d9-4145-bea6-ee5e83bba04e.png">
> 
> Thanks. Overall this implementation is really nice and will help a lot for tablet compatibility.

Thanks @rt4914 fixed above 3 points. For last 3rd point I fixed it by removing justification. Is there any other method to justify the text?

Reply:
No, I checked with mocks it is not justified. I will leave it as is.